templates/base.html

```diff
@@ -18,6 +18,13 @@
 
     <link rel="canonical" href="https://{{SITE_DOMAIN}}{{request.path}}" />
 
+    <link rel="apple-touch-icon" sizes="76x76" href="/apple-touch-icon.png">
```

Comment:
`href="/apple-touch-icon.png">` these hrefs are all broken when you have it in a subdirectory

Reply:
They aren't broken, I created routes for everything. Roots point to root, but my resources are in the folder.

Reply:
if they're not in `output/` then they're broken